pkg/gds/emails/client.go

```diff
@@ -49,14 +51,19 @@ func New(conf config.EmailConfig) (m *EmailManager, err error) {
 // EmailManager allows the server to send rich emails using the SendGrid service.
 type EmailManager struct {
 	conf         config.EmailConfig
-	client       *sendgrid.Client
+	Client       EmailClient
 	serviceEmail *mail.Address
 	adminsEmail  *mail.Address
 }
 
+// EmailClient is an interface that can be implemented by SendGrid email clients.
+type EmailClient interface {
+	Send(email *sgmail.SGMailV3) (*rest.Response, error)
+}
+
 func (m *EmailManager) Send(message *sgmail.SGMailV3) (err error) {
 	var rep *rest.Response
-	if rep, err = m.client.Send(message); err != nil {
+	if rep, err = m.Client.Send(message); err != nil {
 		return err
 	}
```

Comment:
This turned out to be pretty straight forward, I was worried there were other `sendgrid.Client` messages I was missing!